packages/components/action-link/README.md

```diff
@@ -1,48 +1,15 @@
 # Action link
 
-## Guidance
+## Installation
 
-Find out more about the action link component and when to use it in the [NHS digital service manual](https://service-manual.nhs.uk/design-system/components/action-link).
+See the [main README quick start guide](https://github.com/nhsuk/nhsuk-frontend#quick-start) for how to install this component.
```

Comment:
Could drop this para? Not sure it’s necessary - the instructions on how to install components individually is quite easy to find in the parent README, and this is quite an unusual thing to do anyway?